portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSingleSelectionField.testcase

```diff
@@ -73,13 +73,65 @@ definition {
 	}
 
 	@description = "Verify that a Single Selection Field can be required"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanBeRequired {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-132192 CanBeRequired pending implementation
+		ProductMenu.gotoPortlet(
+                        category = "Content &amp; Data",
+                        portlet = "Forms");
+
+                FormsAdminNavigator.gotoAddForm();
+
+                Form.editName(formName = "Form 1");
+
+                Form.gotoAddField(
+                        fieldPositionNumber = "0",
+                        fieldType = "Single Selection",
+                        formPageNumber = "0");
+
+		var fieldSingleSelectionField = FormFields.getFieldName();
+
+                FormFields.enableSwitcher(fieldName = "Required Field");
+
+                Form.save();
+
+                Form.publishForm();
+
+                FormsAdminNavigator.gotoPublishCopyURLLink();
+
+                var publishedFormPublicURL = Form.getPublishedFormPublicURL(
+                        publishedFormPublicURL = "${publishedFormPublicURL}");
+
+                Navigator.openSpecificURL(url = "${publishedFormPublicURL}");
```

Comment:
Hi @leticiachiu,

You can also use `FormsAdminNavigator.gotoPublishedForm()` right after publishing the form and this macro will follow the same steps - copy the URL, place it in a variable and open it in a new tab.
It will save some lines from your code :)

I'd like to also encourage you to also place a `FormPortlet.waitFormRenderer(text = "Submit")`.
Once we're opening a new tab, there is a chance that any delay occurs on server side and this might lead into a flake. This line would ensure the test is going to wait for Submit button to be rendered before moving on with next steps.